third_party/terraform/resources/resource_bigquery_table.go.erb

```diff
@@ -463,6 +512,10 @@ func resourceTable(d *schema.ResourceData, meta interface{}) (*bigquery.Table, e
 		table.TimePartitioning = expandTimePartitioning(v)
 	}
 
+	if v, ok := d.GetOk("range_partitioning"); ok {
```

Comment:
can you also gate any rangeParitioning additions and the added functions with beta flags? sorry, I wasn't explicit - otherwise we'll be setting it/go lint might complain